qemu/tests/block_repeat_blockdev_add.py

```diff
@@ -0,0 +1,24 @@
+"""Repeatedly blockdev_add/del iothread enabled node"""
+import os
+from avocado.utils import process
+
+from virttest import error_context
+from virttest import data_dir
+
+
+# This decorator makes the test function aware of context strings
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Repeatedly blockdev_add/del iothread enabled node
+
+    1) Run script test on host. The script mainly execute repeatedly
```

Comment:
`Host_RHEL.m9.u1.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.1.0.aarch64.io-github-autotest-qemu.block_repeat_blockdev_add.arm64-pci: PASS`

At present, the shell script can be passed on the arm. Do we have any plans to support this test in the framework in the future?